apps/docs.blocksense.network/app/docs/contracts/overview/page.mdx

```diff
@@ -8,17 +8,17 @@ import { Callout } from "nextra/components";
 
 # Introduction
 
-This document provides a quick-start guide to smart contracts as part of our Blocksense protocol. You can explore the structure and functionality of these contracts and unlock a deeper understanding of our cutting-edge data feed storage mechanism.
+This document provides a quick-start guide to smart contracts as part of our Blocksense protocol. Here you can get a high-level overview of the architecture and the role of smart contracts in the Blocksense network.
 
 ## Folder Structure
 
-Smart `contracts` for Blocksense network are stored inside our public GitHub repository. The contracts folder is organized as presented below:
+Smart contracts for Blocksense network are stored inside our public GitHub repository. The `contracts` folder is organized as presented below:
```

Comment:
nit: Here on `contract` we can add link to https://github.com/blocksense-network/blocksense/tree/main/libs/ts/contracts/contracts